src/tests/ftest/aggregation/aggregation_basic.yaml

```diff
@@ -0,0 +1,50 @@
+hosts:
+    test_servers:
+        - server-A
+        - server-B
+    test_clients:
+        - client-A
+        - client-B
+        - client-C
+        - client-D
+        - client-E
+        - client-F
+timeout: 1200
+server_config:
+    name: daos_server
+    servers:
+        log_mask: INFO
+        bdev_class: nvme
+        bdev_list: ["aaaa:aa:aa.a","bbbb:bb:bb.b"]
+        scm_class: dcpm
+        scm_list: ["/dev/pmem0"]
+pool:
+    mode: 146 # 146 is RW
+    name: daos_server
+    scm_size: 30000000000
+    nvme_size: 300000000000
+    svcn: 1
+    control_method: dmg
+container:
+    type: POSIX
+    control_method: daos
+ior:
+    client_processes:
+        np_12:
+            np: 12
+    test_file: daos:testFile
+    repetitions: 1
+    daos_destroy: False
+    iorflags:
+        ssf:
+          flags: "-v -w -W -r -R -k"
+          ior_api:
+            posix:
+              api: DAOS
+          transfersize_blocksize:
+            1M:
+              transfer_size: '1M'
+              block_size: '16G'
+          objectclass:
+            oclass_SX:
+              daos_oclass: "SX"
```

Comment:
Can we try different class apart from "SX"? eg: RP_GPX.

Reply:
Sure, I'll try that !